### ISSUE TITLE
Replace zodios

### ISSUE THREAD

**Issue by fhennig** [loculus-project/loculus#3305]
As Cornelius reported here: https://github.com/loculus-project/loculus/issues/3295, Zodios seems to be unmaintained, and we're now behind a major version on react-query: https://github.com/loculus-project/loculus/pull/3294.

**Comment by fengelniederhammer** [comment]
We could simply build our own wrapper around axios with the zod types that we have? It's a bit of manual effort, but in the end probably easier to maintain than the complicated zodios stuff.

**Comment by fhennig** [comment]
Yes, @corneliusroemer also said so on Slack. I think it sounds like a good way to solve this. From what I could tell, it shouldn't be too much boilerplate to write.

**Comment by corneliusroemer** [comment]
There might be some less heavy wrappers as well, something like: https://github.com/mattpocock/zod-fetch

**Comment by fengelniederhammer** [comment]
Seems like this is using `fetch`. Personally, I like `axios` better than fetch (mostly because the errors are better)
